### ISSUE TITLE
ChoiceGroup: WCAG issues

### ISSUE THREAD

**Issue by kristiina-richards** [TEHIK-EE/tedi-design-system#51]
### Description of the problem

JAWS reads out checkbox and radio button labels incorrectly - first the checkbox and the the label. Should be read only once not as 2 separate items.  
**Requirement:** [link](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html)

**Comment by kasparsuvi1** [comment]
Breaks check/radio selection by click on the label. Change reverted
